pkgs/development/php-packages/smbclient/default.nix

```diff
@@ -0,0 +1,13 @@
+{ buildPecl, lib, pkgs }:
+buildPecl {
+  pname = "smbclient";
+  version = "1.0.4";
+  sha256 = "07p72m5kbdyp3r1mfxhiayzdvymhc8afwcxa9s86m96sxbmlbbp8";
+
+  LIBSMBCLIENT_INCDIR = "${pkgs.samba.dev}/include/samba-4.0";
+
+  nativeBuildInputs = [ pkgs.pkg-config ];
+  buildInputs = [ pkgs.samba.dev ];
```

Comment:
```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [ samba.dev ];
```

Reply:
Also should just be `samba`, since we want a library here. (Though it will work either way because inputs ignore the derivation outputs.